packages/shopify-api/lib/session/session.ts

```diff
@@ -75,19 +75,46 @@ export class Session {
     return obj;
   }
 
+  /**
+   * The unique identifier for the session.
+   */
   readonly id: string;
+  /**
+   * The Shopify shop domain.
```

Comment:
Could we show an example here? 
example.myshopify.com

I know confusion can happen when shops have custom domains